modules/ROOT/pages/http-caching-policy.adoc

```diff
@@ -6,241 +6,419 @@ endif::[]
+The cache works as a dictionary, where each response stored in the cache is associated with a string called _key_. For example, the expression #[attributes.headers['key']] uses the header called `key` as the entry key.
 
-For example: `#[attributes.headers['key']]`
+=== Cache Size and Entry Expiration
 
-For information on DataWeave variables, see xref:4.1@mule-runtime::dataweave-variables-context.adoc[DataWeave Variables for Mule Runtime] and xref:4.1@mule-runtime::about-mule-message.adoc[Mule Message Structure].
+The cache can hold a specified number of entries at any given time. You can configure this number using the `Maximum Cache Entries` property. Each stored entry is held in the cache memory for a specific period of time, after which the entry expires and must be processed again. This expiration time is called _time to live_ (TTL).
 
-If the evaluation of the expression results in an error due to missing parameters or any other cause, this policy is not applied and passes the request to the next receptor in the policy chain.
+Another condition that can trigger the expiration of an entry is when the cache reaches the maximum number of items that it can store. When this scenario occurs, the entry is removed by using the FIFO (First-In First-Out) criteria. This means that the earliest entry to reach the cache is removed, even though it hasn't reached the TTL value yet.
 
-*Note:*  This parameter is mandatory and it’s default value is `#[attributes.requestPath]`, which means that the key for the caching is the request’s path. If you call the endpoint as `+http://myAppUrl.com/my/policy+` the key is `/my/policy`.
+=== Distributed Cache
 
-=== Maximum Cache Entries
+Each entry in the cache can be shared between different nodes in a cluster or between several workers in Runtime Manager using the `Distributed` option (see table). If this option is not enabled, then each node is assigned its own cache memory.
 
-This is the maximum number of responses that can be present in the cache at the same time. The criteria for removing the responses from the cache when this maximum is surpassed is FIFO. This configuration is ignored in Runtime Manager.
+=== Persistent Cache
 
-This field is also mandatory and has a default value of 10000 entries.
+A persistent cache enables the stored entries in the cache to persist if the Mule runtime engine (Mule) is restarted.
 
-*Note:* When using the Runtime Manager ObjectStore v2, this is not enforced.
+When you upgrade a Mule version that has the HTTP Caching policy configured to use the persistent store, the policy tries to maintain the entries stored by the previous Mule version. However in a worst-case scenario, the entries in the cache are invalidated and the cache is re-populated when new requests arrive. This manipulation of the entries in the cache occurs automatically and is invisible to the user.
+//When you upgrade to a Mule version?--Lorena
```

Comment:
In CloudHub, for example. You can upgrade your runtime instance manually.

Reply:
Ok Thanks!